zabbix/umbrel-app.yml

```
@@ -0,0 +1,39 @@
+manifestVersion: 1.1
+id: zabbix
+category: networking
+name: Zabbix
+tagline: The all-in-one, open-source solution that lets you monitor anything
+icon: https://raw.githubusercontent.com/deralia/umbrel-app-store/master/deralia-zabbix/assets/icon.png
+version: "7.0.3"
+description: >-
+  A Zabbix Server + Frontend interface to monitor your network.
+
+
+  The first time running Zabbix might take a few minutes because it needs to initialize the database. Please be patient.
+
+
+  The Zabbix Agent listens on port 10050, the Zabbix Server listens on port 10051 and Zabbix Frontend listens on port 10052.
+
+
+  If you want to self-monitor your Umbrel OS, change the Zabbix Server IP Address (127.0.0.1) and use the DNS instead (the DNS name should be "zabbix-agent").
+developer: Zabbix LLC
+website: https://www.zabbix.com/
+dependencies: []
+repo: https://github.com/zabbix/zabbix
+support: https://www.zabbix.com/support
+port: 10052
+gallery:
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/01.png
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/02.png
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/03.png
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/04.png
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/05.png
+  - https://raw.githubusercontent.com/getumbrel/umbrel-apps/master/zabbix/assets/06.png
+path: "/admin/"
```

This makes it so that when a user clicks on the Zabbix icon they are taken to `http://umbrel.local:10052/admin/`. But this path results in a 404 response. I haven't tested, but you can likely just change this to 
```suggestion
path: ""
```

---

Nice catch! That was here because I used Pi-Hole's settings as the template file for me. Pushing now!